lib/checkIssueAssignee.js

```diff
@@ -16,74 +16,71 @@
  * @fileoverview Handler to check issue assigned event.
  */
 
-const core = require('@actions/core');
-const { context, GitHub } = require('@actions/github');
 const { google } = require('googleapis');
+const apiForSheets = require('./apiForSheets');
 
-const checkAssignees = async () => {
-  core.info('A USER GOT ASSIGNED TO AN ISSUE...');
-  const token = core.getInput('repo-token');
-  const octokit = new GitHub(token);
+ /**
+ * Checks that a user has signed the CLA.
+ *
+ * @param {String} username - Username of the user that added the label.
+ */
+const hasSignedCla = async (username) => {
+  const clientSecret = process.env.CLIENT_SECRET;
+  const CLA_SHEET_ID = process.env.SPREADSHEET_ID;
+  const oauthClient = apiForSheets.authorize(JSON.parse(clientSecret))
+  const sheets = google.sheets('v4');
+
+  const rows = await sheets.spreadsheets.values.get({
+    auth: oauthClient,
+    spreadsheetId: CLA_SHEET_ID,
+    range: 'Usernames!A:A',
+  });
+
+  const hasUserSignedCla = rows.data.values.some(row => {
+    return row[0].toLowerCase() === username.toLowerCase();
+  });
+
+  return hasUserSignedCla;
+};
+
+
+/**
+ * @param {import('probot').Context} context
+ */
+const checkAssignees = async (context) => {
   const assignee = context.payload.assignee;
   const issue = context.payload.issue;
   const linkToCla = 'here'.link(
     'https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#setting-things-up');
 
   // Catch and log any error that will occur while accessing google sheets api.
   try {
-    core.info('Checking if ' + assignee.login + ' has signed the CLA');
+    console.log('Checking if ' + assignee.login + ' has signed the CLA');
     const assigneeHasSignedCla = await hasSignedCla(assignee.login);
     if (!assigneeHasSignedCla) {
-      core.info(assignee.login + ' has not signed the CLA');
+      console.log(assignee.login + ' has not signed the CLA');
 
       const commentBody = 'Hi @' + assignee.login + ', you need to sign the ' +
```

Comment:
Add another line in the comment: I am unassigning you for now, feel free to assign yourself once you have signed the CLA.

Reply:
Done.